packages/contracts/test/LibGuiseLevel.t.sol

```diff
@@ -0,0 +1,54 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.0;
+
+import { MudV2Test } from "@latticexyz/std-contracts/src/test/MudV2Test.t.sol";
+import { LibGuiseLevel } from "../src/guise/LibGuiseLevel.sol";
+import { LibExperience } from "../src/charstat/LibExperience.sol";
+import { Experience, ExperienceTableId } from "../src/codegen/Tables.sol";
+import { ActiveGuise, ActiveGuiseTableId } from "../src/codegen/Tables.sol";
+import { GuisePrototype, GuisePrototypeTableId } from "../src/codegen/Tables.sol";
+import { PStat_length } from "../src/CustomTypes.sol";
+
+contract LibGuiseLevelTest is MudV2Test {
+  bytes32 internal targetEntity = keccak256("targetEntity");
+
+  function setUp() public virtual override {
+    super.setUp();
+    vm.startPrank(worldAddress);
+  }
+
+  function testGetAggregateLevel() public {
+    // Initialize exp and levelMul
+    LibExperience.initExp(targetEntity);
+    assertTrue(LibExperience.hasExp(targetEntity));
```

Comment:
I think it's fine to put this into `setUp`, since you test exp in `LibExperienceTest`